### ISSUE TITLE
Snake Skin Boot needs a buff.

### ISSUE THREAD

**Issue by QuirkySmirkyIan** [jasonrohrer/OneLife#815]
The snake skin boot (766) and fruit boot (844) only provide 5% insulation. However the wooden shoe (2886) provides 6% insulation and rabbit fur shoe (203) provides 8.5% insulation. As the snake boots decay and are significantly harder to acquire then the other shoes especially rabbit shoes as they are a specialty item and run the risk of a really bad animal wound I suggest raising the snake skin boot / fruit boot insulation value to 10%.

TLDR: Snake boot is high risk very low reward and the insulation value should go from 5% to 10%.
